src/renderer/Instance.ts

```diff
@@ -19,11 +19,12 @@ const app = remote.app;
 export type ModalType = "options" | "rename" | "saves" | "delete" | "logs";
 export default class Instance implements InstanceData {
 	public static readonly MINECRAFT_PATH = path.join(app.getPath("userData"), "./game/");
+	public static readonly SAVE_PATH = path.join(app.getPath("userData"), "./instances/");
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=6041480)